### PR TITLE
Update chinook and nventive packages

### PR DESCRIPTION
**Pull request by lamonfly**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [X] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj [hunk 1]
```diff
@@ -1,254 +1,254 @@
...
+		<PackageReference Include="Uno.UI" Version="4.4.20" />
+		<PackageReference Include="Uno.UI.RemoteControl" Version="4.4.20" />
+		<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.36" />
+		<PackageReference Include="Chinook.DynamicMvvm.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.SectionsNavigation.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.DataLoader.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.BackButtonManager.Uno" Version="1.1.1" />
+		<PackageReference Include="Nventive.AsyncWebView.Uno" Version="0.4.0-dev.22" />
+		<PackageReference Include="Nventive.ExtendedSplashScreen.Uno" Version="0.3.0-dev.38" />
+		<PackageReference Include="Nventive.Persistence.Uno" Version="0.2.0-feature.uno-ui-4.21" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:125]: Is that really the version you made? feature uno-ui-4 ?

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj [hunk 2]
```diff
@@ -1,254 +1,254 @@
...
-      <Name>ApplicationTemplate.Client</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\ApplicationTemplate.Presentation\ApplicationTemplate.Presentation.csproj">
-      <Project>{ab67372c-6cf2-46a5-bc86-7cd257af3ff0}</Project>
-      <Name>ApplicationTemplate.Presentation</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="..\ApplicationTemplate.Shared.Views\ApplicationTemplate.Shared.Views.projitems" Label="Shared" />
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
+	<PropertyGroup>
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:3]: do we really have to change all lines?

Comment (jeanplevesque) [reply src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:3]: It adds a lot of noise for the review.

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj [hunk 3]
```diff
@@ -1,254 +1,254 @@
...
+		<PackageReference Include="Uno.UI.RemoteControl" Version="4.4.20" />
+		<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.36" />
+		<PackageReference Include="Chinook.DynamicMvvm.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.SectionsNavigation.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.DataLoader.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.BackButtonManager.Uno" Version="1.1.1" />
+		<PackageReference Include="Nventive.AsyncWebView.Uno" Version="0.4.0-dev.22" />
+		<PackageReference Include="Nventive.ExtendedSplashScreen.Uno" Version="0.3.0-dev.38" />
+		<PackageReference Include="Nventive.Persistence.Uno" Version="0.2.0-feature.uno-ui-4.21" />
+		<PackageReference Include="Nventive.View.Uno" Version="0.4.0-feature.dotnet6.50" />
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:126]: There are still updates being made for that package, no? This update should wait for the real release. Same comments apply to the packages that use feature-branch versions. Just finish the work on the packages and use the ones generated from the main branch.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj (hunk 3 above)
Comment (GabrielChouinardLetourneau) [reply src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:126]: I thought that this PR was to make sure the changes we did on the updated packages won't break the current template for Uno. We could change the target branch to a feature one to make sure main won't be affected

Comment (jeanplevesque) [reply src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj]: The goal of the PR is good. Just make it a Draft PR until all packages are finished :)

File: src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj [hunk 4]
```diff
@@ -103,27 +103,27 @@
...
+    <PackageReference Include="Chinook.SectionsNavigation.Uno" Version="1.1.0" />
+    <PackageReference Include="Chinook.DataLoader.Uno" Version="1.1.0" />
+    <PackageReference Include="Chinook.BackButtonManager.Uno" Version="1.1.1" />
     <PackageReference Include="Nventive.AsyncWebView.Uno" Version="0.4.0-dev.22" />
-    <PackageReference Include="Nventive.ExtendedSplashScreen.Uno" Version="0.3.0-dev.33" />
-    <PackageReference Include="Nventive.Persistence.Uno" Version="0.2.0-dev.17" />
-    <PackageReference Include="Nventive.View.Uno" Version="0.4.0-dev.49" />
+    <PackageReference Include="Nventive.ExtendedSplashScreen.Uno" Version="0.3.0-dev.38" />
+    <PackageReference Include="Nventive.Persistence.Uno" Version="0.2.0-dev.26" />
+    <PackageReference Include="Nventive.View.Uno" Version="0.4.1-feature.dotnet6.53" />
```
Comment (carlh98) [inline src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj:126]: @lamonfly  please use [0.4.1-dev.51](https://www.nuget.org/packages/Nventive.View.Uno/0.4.1-dev.51)

**Review by jeanplevesque — APPROVED** [review]

**Review by takla21 — APPROVED** [review]

**Review by carlh98 — APPROVED** [review]

**Review by GabrielChouinardLetourneau — APPROVED** [review]

**Pull request merged**